it/it-plugins/billing-plugin/src/main/java/BillingValidations.java

```diff
@@ -27,6 +27,7 @@
 public class BillingValidations implements BillingValidationsExtension {
 
   private static final String PREVENT_PROJECT_ANALYSIS_SETTING = "sonar.billing.preventProjectAnalysis";
+  private static final String PREVENT_UPDATING_PROJECTS_VISIBILITY_TO_PRIVATE_SETTING = "sonar.billing.preventUpdatingProjectsVisibilityToPrivate";
```

Comment:
It's useless these properties are defined in the fake plugin, which is visible by no one